recipes/scikit-video/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "scikit-video" %}
+{% set version = "1.1.11" %}
+{% set sha256 = "5061d2aeae1892b73a97c89a82942b3e8eebf2fe23e59c60e06ede5f8a24ed1e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
```

Comment:
It looks like this builds against `numpy`, if so please take a look at:

https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
I'm not sure what "links against numpy" means exactly, but looking at https://github.com/scikit-video/scikit-video/blob/master/setup.py I only see a `from numpy.distutils.core import setup`, which from the docs doesn't seem to be a "link" case. What do you think?

Reply:
It is creating a compiled extension, right? If so it is probably safer to use the "building against numpy" instructions.

Reply:
That is a common things when building `numpy` extensions if I remember correctly the `numpy` import monkey patches the `setuptools`.

Reply:
😕 doesn't seem so: https://github.com/numpy/numpy/blob/master/doc/DISTUTILS.rst.txt#id4

Reply:
You can raise it upstream but my fuzzy memory tells me that is correct. Although things may have changed lately.

Reply:
ok thanks, sorry for the many questions...

If you think numpy pinning is ok now you can merge and archive the old sk-video feedstock when CI passes.

Reply:
> ok thanks, sorry for the many questions...

No problem.

> If you think numpy pinning is ok now you can merge and archive the old sk-video feedstock when CI passes.

Done!